interfaces/builtin/content_test.go

```diff
@@ -987,10 +987,6 @@ slots:
 	expectedMnt := []osutil.MountEntry{{
 		Name:    "/var/snap/producer/2/directory",
 		Dir:     "/var/snap/consumer/common/import/directory",
-		Options: []string{"bind", "ro"},
-	}, {
-		Name:    "/var/snap/producer/2/directory",
-		Dir:     "/var/snap/consumer/common/import/directory-2",
```

Comment:
does this happen because in this test what is the same? I see there is still one test with -2 behavior? do we need more of them?

Reply:
Yes, these two entries are now getting merged. I also noticed that the clash code path was still triggered by another test, so I didn't see a need to add a new one.